packages/strapi-database/lib/queries/create-query.js

```diff
@@ -52,15 +53,21 @@ class Query {
   }
 
   async create(...args) {
-    return this.connectorQuery.create(...args);
+    const entry = await this.connectorQuery.create(...args);
```

Comment:
`🤔 question` at this moment, are we sure the data format is the same for Bookshelf and Mongoose?
No Bookshelf of Mongoose instance?